crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -80,13 +79,8 @@ fn main() -> Result<()> {
     let pulse_path = pulse_path.join("native");
     setup_socket_proxy(pulse_path, 3333)?;
 
-    if !setup_x11_forwarding(run_path)? {
-        // Will not return if successful.
-        exec_sommelier(&options.command, &options.command_args)
-            .context("Failed to execute sommelier")?;
-    }
+    setup_x11_forwarding(run_path)?;
```

Comment:
This should be changed to return `Result<()>` and change the false return case to an error.